scripts/provision.sh

```diff
@@ -229,25 +275,152 @@ start_builder() {
   if [ "${ARTIFACTORY_ENABLED:-false}" = "false" ] && [ "${S3_ENABLED:-false}" = "false" ]; then
```

Comment:
It's not part of this pr, but should these also be `==`?

Reply:
Nope. As I understand it, the `[]` in bash are a synonym for `test` and for the `test` syntax this is correct.